### PR TITLE
moved authorization param list to constants in const.py

### PR DESCRIPTION
**Pull request by yashLadha**

### CODE REVIEW

**Comment by yashLadha** [comment]
@nworden @gimite

**Comment by nworden** [comment]
Hi Yash,

I'm not sure I understand the motivation for moving this list to `const.py`. `const.py` is meant for values that are shared between multiple modules or handlers (for example, `ROOT_URL` is needed in multiple places across the codebase, so there's no other single file that would be a particularly good place for it). However, the authorization param list is only used in `admin_api_keys.py`, so I don't think we need to move it to `const.py`.

All that said, I definitely agree that it would be better for it to be a constant instead of being embedded in the function. How about just moving it back from `const.py` (and putting it alongside `API_KEY_LENGTH` and `KEYS_PER_PAGE` in `admin_api_keys.py`)?

**Comment by yashLadha** [comment]
Yaa sounds good @nworden  , updating it

File: app/admin_api_keys.py [hunk 1]
```diff
@@ -28,23 +28,25 @@
 API_KEY_LENGTH = 16
 KEYS_PER_PAGE = 50
 
+# Authorization params list
```
Comment (nworden) [inline app/admin_api_keys.py]: Nit: I think you can delete this comment. It's not needed since it's redundant with the name of the constant itself.

**Comment by yashLadha** [comment]
Done @nworden

File: app/admin_api_keys.py (hunk 1 above)
Comment (yashLadha) [reply app/admin_api_keys.py]: :+1:

**Comment by yashLadha** [comment]
Done @nworden

**Pull request merged**